onyo/lib/commands.py

```diff
@@ -254,48 +245,40 @@ def _edit_asset(inventory: Inventory,
                 asset: dict | UserDict,
                 operation: Callable,
                 editor: str | None) -> dict | UserDict:
-    r"""Edit `asset` via configured editor and a temporary asset file.
+    r"""Edit an ``asset`` (as a temporary file) with ``editor``.
+
+    A helper for ``onyo_edit()`` and ``onyo_new(edit=True)``.
 
-    Utility function for `onyo_edit` and `onyo_new(edit=True)`.
-    This is editing a temporary file initialized with `asset`. Once
-    the editor is done, `asset` is updated from the file content and
-    `operation` is tried in order to validate the content for a
-    particular purpose (Currently used: Either `Inventory.add_asset`
-    or `Inventory.modify_asset`).
-    User is asked to either keep editing or accept the changes
-    (if valid).
+    The asset content is edited as a temporary file. Once the editor is done,
+    the ``operation`` function is executed to validate the content.
+
+    The user is then presented with a diff and prompted on how to proceed
+    (accept, edit, skip, or abort).
 
     Parameters
     ----------
     inventory
-      Inventory to edit `asset` for. This is primarily used to check
-      whether `operation` resulted in registered operations with that
-      inventory in order to remove them, if the edit was not accepted.
+        The Inventory containing the asset to edit.
     asset
-      Asset to edit.
-    editor
-      Editor to use. This is a to-be executed shell string, that gets
-      a path to a temporary file. Defaults to `OnyoRepo.get_editor()`.
+        The asset to edit.
     operation
-      Function to call with the resulting asset. This function is
-      expected to raise, if the edited asset isn't valid for that
-      purpose.
-
-    Returns
-    -------
-    dict
-      The edited asset.
+        Function to validate the edited asset, which shall raise if the asset
+        isn't valid for that purpose.
+        In practice, see ``inventory.py``.
```

Comment:
"In practice" sounds super weird. No idea what it should tell me, or how it would be different "in theory".
Maybe "see ``inventory.py``." would be enough, or if this is important, "In practice" should be replaced for something more explanatory.

Reply:
I have replaced this with a direct reference to an example Operator.